groovy/gradle/src/org/netbeans/modules/gradle/execute/GradleDaemonExecutor.java

```diff
@@ -69,6 +69,9 @@
     private static final Logger LOGGER = Logger.getLogger(GradleDaemonExecutor.class.getName());
 
     private final ProgressHandle handle;
+    private OutputStream outStream;
```

Comment:
So, assuming the cancel can be asynchronous, I wonder if there's a need for synchronization for these fields?